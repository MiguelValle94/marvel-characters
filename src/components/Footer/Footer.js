import React from 'react';

import './Footer.scss';

export default function Footer() {
  return (
    <div className='footer bg-light'>
      Marvel Characters - Powered by jlsanchezsanz
    </div>
  );
}
