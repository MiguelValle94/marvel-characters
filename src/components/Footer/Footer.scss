@import '../../styles/variables.scss';

.footer {
  margin-top: var(--space-4);
  padding: var(--space-2);
}
