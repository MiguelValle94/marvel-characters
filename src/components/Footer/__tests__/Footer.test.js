import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../Footer';

const setUpShallow = () => {
  const component = shallow(<Footer />);
  return component;
};

describe('Footer', () => {
  it('should match snapshot', () => {
    const component = setUpShallow();
    expect(component).toMatchSnapshot();
  });
});
