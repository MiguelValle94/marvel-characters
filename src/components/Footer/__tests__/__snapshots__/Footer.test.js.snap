// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<div
  className="footer bg-light"
>
  Marvel Characters - Powered by jlsanchezsanz
</div>
`;
